cmd/skaffold/app/cmd/deploy_test.go

```diff
@@ -24,26 +24,32 @@ import (
 	"testing"
 
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/build"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest"
 	"github.com/GoogleContainerTools/skaffold/testutil"
 )
 
 func TestRunDeploy(t *testing.T) {
```

Comment:
Not introduced by your change - but did we notice that this test actually a copy paste of the build tests and has nothing to do with Deploy? :D